index.html

```diff
@@ -3619,13 +3619,13 @@ <h3>
 
     <ol class="punctuation" type="1">
       <li id="id202">
-        <p its-locale-filter-list="en" lang="en" class="translateme">Translation needed.</p>
+        <p its-locale-filter-list="en" lang="en">Brackets need rotate 90 degree clockwise in vertical writing mode.</p>
         <p its-locale-filter-list="zh-hans" lang="zh-hans">夾注符号于直排时，需顺时针旋转90°。</p>
         <p its-locale-filter-list="zh-hant" lang="zh-hant">夾注符號於直排時，需順時針旋轉90°。</p>
       </li>
 
       <li id="id203">
-        <p its-locale-filter-list="en" lang="en" class="translateme">Translation needed.</p>
+        <p its-locale-filter-list="en" lang="en">Brackets always come in pairs. The front part called opening brackets can't appear at the end of a line. The latter part called closing brackets can't appear at the begining of a line.</p>
```

Comment:
Perhaps: 'the front part called opening brackets' -> 'the opening bracket', and likewise for the closing bracket.
begining -> beginning